lib/pdf/pdf.go

```diff
@@ -0,0 +1,98 @@
+package pdf
+
+import (
+	"bytes"
+	"math"
+	"strings"
+
+	"github.com/jung-kurt/gofpdf"
+)
+
+type GoFPDF struct {
+	pdf *gofpdf.Fpdf
+}
+
+func Init() *GoFPDF {
+	newGofPDF := gofpdf.NewCustom(&gofpdf.InitType{
+		UnitStr:    "in",
+		FontDirStr: "./d2renderers/d2fonts/ttf",
```

Comment:
this won't work

Reply:
hmmm so far this is the only path that works for me lol.

Reply:
??

i don't understand. this is a string. at runtime this'll look for this path from wherever they're running d2